### ISSUE TITLE
need solution for referencing prev day from workflow definition

### ISSUE THREAD

**Issue by akonopko** [collectivemedia/celos#179]
in sibyl-google workflow definition there is couple of places where "previous day" is referenced
for example: 
1. "sqoopDirectory" variable in sqoop workflow
it is path which is constructed basing on the date of previous day
it is used from variety of places from inside the workflow definition so there is no simple way to get this data
1. same "extractPath" variable 
2. also year, month, day variables which are expected for workflow should be for previous day

Basically thats all what is needed for finishing sibyl migration now
